Source/SwiftyJSON.swift

```diff
@@ -58,14 +58,14 @@ public struct JSON {
     
     /**
      Creates a JSON using the data.
-     
-     - parameter data:  The NSData used to convert to json.Top level object in data is an NSArray or NSDictionary
+
+     - parameter data:  The NSData used to convert to json. Top level object in data is an NSArray or NSDictionary
      - parameter opt:   The JSON serialization reading options. `.AllowFragments` by default.
-     - parameter error: error The NSErrorPointer used to return the error. `nil` by default.
-     
+     - parameter error: The NSErrorPointer used to return the error. `nil` by default.
+
      - returns: The created JSON
      */
-    public init(data:Data, options opt: JSONSerialization.ReadingOptions = .allowFragments, error: NSErrorPointer = nil) {
+    public init(data:NSData, options opt: NSJSONReadingOptions = .AllowFragments, error: NSErrorPointer = nil) {
```

Comment:
We are currently supporting swift 3 and Xcode 8 on master, this change reverted part of that effort. :smile: